data/requirements.txt

```diff
@@ -1,7 +1,8 @@
 ipapy>=0.0.9.0
+lxml_html_clean
```

Comment:
Were you trying to use lxml_html_clean somewhere? It's not used in this pull request.

Reply:
See note here: 

https://github.com/CUNY-CL/wikipron/pull/529/commits/21054c3be6227f8fb7184ab9194f42465f03107a

Tests fail without it, and including it is the suggested solution.